packages/apple-targets/src/withXcodeChanges.ts

```diff
@@ -652,7 +655,7 @@ function createConfigurationList(
       GCC_C_LANGUAGE_STANDARD: "gnu11",
       GENERATE_INFOPLIST_FILE: "YES",
       INFOPLIST_FILE: cwd + "/Info.plist",
-      INFOPLIST_KEY_CFBundleDisplayName: name,
+      INFOPLIST_KEY_CFBundleDisplayName: displayName,
```

Comment:
Heads up, you should be able to just set this key in your targets/<target>/Info.plist (CFBundleDisplayName) to override the default here. Would be nice if we could just have one `name` property in the config.